src/gbxml/ReverseTranslator.cpp

```diff
@@ -1046,6 +1020,30 @@ namespace gbxml {
     return result;
   }
 
+  boost::optional<openstudio::model::ModelObject> ReverseTranslator::translateId(const pugi::xml_node& element,
+                                                                                 openstudio::model::ModelObject& modelObject) {
+    std::string id = element.attribute("id").value();
+    m_idToObjectMap.insert(std::make_pair(id, modelObject));
+    modelObject.setName(id);
+
+    model::AdditionalProperties result = modelObject.additionalProperties();
+    result.setFeature("gbXMLId", id);
+
+    return result;
+  }
+
+  boost::optional<openstudio::model::ModelObject> ReverseTranslator::translateName(const pugi::xml_node& element,
+                                                                                   openstudio::model::ModelObject& modelObject) {
+    model::AdditionalProperties result = modelObject.additionalProperties();
+
+    auto name = element.child("Name").text();
+    if (!name.empty()) {
+      result.setFeature("displayName", name.as_string());
+    }
+
+    return result;
```

Comment:
use the helper you added no? `bool ModelObject::setDisplayName(const std::string& displayName)`